src/Facility.CodeGen.CSharp/CSharpGenerator.cs

```diff
@@ -910,7 +910,7 @@ private CodeGenFile GenerateHttpMapping(HttpServiceInfo httpServiceInfo, Context
 												foreach (var field in httpMethodInfo.RequestNormalFields)
 												{
 													var fieldName = context.GetFieldPropertyName(field.ServiceField);
-													code.WriteLine($"{fieldName} = (({requestTypeName}) body{NullableReferenceBang}).{fieldName},");
+													code.WriteLine($"{fieldName} = (({requestTypeName}?) body)?.{fieldName},");
```

Comment:
`{requestTypeName}?` is an error when nullable types are not enabled.

Reply:
Thanks! #102